sdk/core/azure-core-amqp/samples/eventhub_async_writer_sample/eventhub_async_writer_sample.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-Licence-Identifier: MIT
+
+#include <azure/core/amqp/connection.hpp>
+#include <azure/core/amqp/message_sender.hpp>
+#include <chrono>
+#include <iostream>
+#include <limits>
+#include <string>
+
+#define EH_CONNECTION_STRING "<<<Replace with the connection string from your eventhubs instance>>>"
+
+int main()
+{
+  auto credentials{
+      std::make_shared<Azure::Core::Amqp::_internal::SaslPlainConnectionStringCredential>(
+          EH_CONNECTION_STRING)};
```

Comment:
Presumably, this is for SDK authors and not end customers, correct? We don't want the `_internal` namespace to show up in end customer/application code.

Reply:
The only part of AMQP that is public is the Models namespace. But we need samples for developers who want to use the AMQP library.